e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs

```diff
@@ -42,9 +42,10 @@ public class RegistryManagerExportDevicesTests : E2EMsTestBase
             JobStatus.Unknown,
         };
 
-        [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
+        [LoggedTestMethod, Timeout(LongRunningTestTimeoutMilliseconds)]
         [TestCategory("LongRunning")]
         [DoNotParallelize] // the number of jobs that can be run at a time are limited anyway
+        [Ignore]
```

Comment:
Why ignore?

Reply:
Cannot run in the matrix tests in the build pipeline run because the [DoNotParallelize] flag do not work in that setup.
We can either skip this test or need to create an separate job to run these type of tests on each framework in sequence.

Reply:
Created a task for us to improve these ignored tests.